libs/RobotKit/tests/RobotController_test_stateAutonomousActivities.cpp

```diff
@@ -124,26 +125,100 @@ TEST_F(RobotControllerTest, stateGameActivityDisplayTagsChosen)
 
 	EXPECT_CALL(mock_videokit, displayImage).Times(1);
 
+	activitykit.start(MagicCard::number_10);
```

Comment:
Since it's setup, I suggest the same lambda func above `set_activitykit_is_playing`